cfgov/unprocessed/css/video-player.less

```diff
@@ -227,3 +227,19 @@
 	   text-align: right;
 	}
 } );
+
+
+// Special styles for playing within an FCM
+.o-featured-content-module {
+  &_visual {
```

Comment:
Minor: Spaces should be 4, not 2

Reply:
Addressed in 2c8bc3e, rebased from master, force pushed.